data/config/TCStatConfig_default

```diff
@@ -134,6 +134,20 @@ init_str_val  = [];
 init_str_exc_name = [];
 init_str_exc_val  = [];
 
+//
+// Stratify track points by applying thresholds to numeric
+// storm diagnostic values in the TCDIAG lines.
+//
+diag_thresh_name = [];
+diag_thresh_val  = [];
```

Comment:
What are valid values that can go here? Thresholds in numeric (e.g. gt) notation only? It would be useful to have an example in a unit test to add support in the METplus wrapper.

Reply:
Please see this newly update [unit test](https://github.com/dtcenter/MET/blob/bc2b5559fd08e8b41e70938a5c7f0e4540a27a98/internal/test_unit/xml/unit_tc_pairs.xml#L191) for an example. `diag_name` is an array of strings defining which diagnostics should be extracted from the the input files.

Reply:
Yes, as we discussed in the 10/27/2022 MET development meeting, here's an example:
```
diag_thresh_name = [ "TPW", "SHRG" ];
diag_thresh_val = [ gt50, le100 ];
```
Marking as resolved.